### PR TITLE
Add ruby_debugger#statusline

### PR DESCRIPTION
**Pull request by ecoffey**
This adds a function for use in the vim statusline.

If the ruby debugger is active the status line will indicate so, otherwise nothing is displayed.

### CODE REVIEW

**Comment by astashov** [comment]
I really can't realize where it is put into vim statusline. :)
Could you please point to it for me?

**Comment by ecoffey** [comment]
Oh sorry, I think I described it wrong.

It doesn't insert it into the statusline, but gives you a clean string that a end user can put into their status line.  Kind of like how tpop/vim-fugitive and vim-rvm work.

File: autoload/ruby_debugger.vim [hunk 1]
```diff
@@ -36,6 +36,13 @@ fun! ruby_debugger#load_debugger()
   endif
 endf
```
Comment (ecoffey) [inline autoload/ruby_debugger.vim:38]: This is the function you would call as part of your status line declaration

``` VimL
set statusline=%{ruby_debugger#statusline()}
```

**Comment by astashov** [comment]
Ah, got it. Could you please add a note about that to doc/ruby_debugger.txt, and I'll merge it.
Thanks!

**Comment by ecoffey** [comment]
Will do, commits incoming soon!

**Comment by ecoffey** [comment]
Bump :-)

**Comment by astashov** [comment]
Whoops, sorry, missed your commit. Merging...
Thanks a lot, Eoin!

**Pull request merged**